cmd/buildkitd/config/config.go

```diff
@@ -75,9 +75,13 @@ type OTELConfig struct {
 }
 
 type GCConfig struct {
-	GC            *bool      `toml:"gc"`
-	GCKeepStorage DiskSpace  `toml:"gckeepstorage"`
-	GCPolicy      []GCPolicy `toml:"gcpolicy"`
+	GC *bool `toml:"gc"`
+	// Deprecated: use GCReservedSpace instead
+	GCKeepStorage   DiskSpace  `toml:"gckeepstorage"`
+	GCReservedSpace DiskSpace  `toml:"reservedSpace"`
+	GCMaxUsedSpace  DiskSpace  `toml:"maxUsedSpace"`
+	GCMinFreeSpace  DiskSpace  `toml:"minFreeSpace"`
```

Comment:
The other toml tags here are all lowercase

Reply:
Yeah, I know. But the `GCPolicy` is camelcase, and I didn't want the same keys to be in different casing within the same toml.